tests/wpt/metadata/webgl/conformance-1.0.3/conformance/textures/tex-image-and-sub-image-2d-with-webgl-canvas-rgba4444.html.ini

```diff
@@ -1,5 +1,7 @@
 [tex-image-and-sub-image-2d-with-webgl-canvas-rgba4444.html]
   type: testharness
+  disabled:
+    if os == "linux": https://github.com/servo/servo/issues/12864
```

Comment:
Oh, right.

I was just autopiloting here.  Given that I haven't actually changed anything about MacOS in this series, we should probably drop this commit and ignore the macos failure.